ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-release-4.17__okd-scos.yaml

```diff
@@ -0,0 +1,36 @@
+base_images:
+  ocp_builder_stream9:
+    name: builder
+    namespace: ocp
+    tag: stream9
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.22-openshift-4.17
+images:
+- build_args:
+  - name: TAGS
+    value: scos
+  dockerfile_path: Dockerfile.ocp
```

Comment:
```suggestion
  dockerfile_path: Dockerfile.okd
```

Reply:
okd or scos? why the difference in some of your suggestions?

Reply:
the [ironic-agent-image](https://github.com/openshift/ironic-agent-image/tree/main) is a different repo from the [ironic-image](https://github.com/openshift/ironic-image)